bokehjs/src/coffee/models/renderers/graph_renderer.ts

```diff
@@ -17,34 +22,34 @@ export class GraphRendererView extends RendererView {
     [this.node_view, this.edge_view] = build_views(this._renderer_views,
       [this.model.node_renderer, this.model.edge_renderer], this.plot_view.view_options());
 
-    this.set_data();
+    this.set_data(true);
   }
 
   connect_signals(): void {
     super.connect_signals();
 
-    this.connect(this.model.layout_provider.change, () => this.set_data())
-    this.connect(this.model.node_renderer.data_source._select, () => this.set_data())
-    this.connect(this.model.node_renderer.data_source.inspect, () => this.set_data())
-    this.connect(this.model.node_renderer.data_source.change, () => this.set_data())
-    this.connect(this.model.edge_renderer.data_source._select, () => this.set_data())
-    this.connect(this.model.edge_renderer.data_source.inspect, () => this.set_data())
-    this.connect(this.model.edge_renderer.data_source.change, () => this.set_data())
+    this.connect(this.model.layout_provider.change, () => this.set_data(true))
+    this.connect(this.model.node_renderer.data_source._select, () => this.set_data(true))
+    this.connect(this.model.node_renderer.data_source.inspect, () => this.set_data(true))
+    this.connect(this.model.node_renderer.data_source.change, () => this.set_data(true))
+    this.connect(this.model.edge_renderer.data_source._select, () => this.set_data(true))
+    this.connect(this.model.edge_renderer.data_source.inspect, () => this.set_data(true))
+    this.connect(this.model.edge_renderer.data_source.change, () => this.set_data(true))
 
     const {x_ranges, y_ranges} = this.plot_model.frame
 
     for (const  name in x_ranges) {
       const rng = x_ranges[name];
-      this.connect(rng.change, () => this.set_data())
+      this.connect(rng.change, () => this.set_data(true))
     }
 
     for (const name in y_ranges) {
       const rng = y_ranges[name];
-      this.connect(rng.change, () => this.set_data())
+      this.connect(rng.change, () => this.set_data(true))
     }
   }
 
-  set_data(request_render) {
+  set_data(request_render: boolean) {
     // TODO (bev) this is a bit clunky, need to make sure glyphs use the correct ranges when they call
     // mapping functions on the base Renderer class
     if (request_render == null) { request_render = true; }
```

Comment:
Instead of sprinkling `true` around the code, this line could be removed and the signature changed to `set_data(request_render: boolean = true)`. However, I'm not sure why all those changes are needed in this PR. If those changes aren't strictly necessary, you are greatly reducing mergeability of this PR this way.

Reply:
Actually, I already fixed this in `master`.